### PR TITLE
Add first version of essay <documentation

### PR DESCRIPTION
**Pull request by jochem725**
Using this pull request I would like to ask for changes or additions to the current version of the essay. 
I tried to write down as much as possible about the process, but I'm sure we can add more things to this reflection.

### CODE REVIEW

**Comment by bryanvanwijk** [comment]
I have added some sentences to the essay.

File: deliverables/sprint5/exercise_3_essay.tex [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+\subsection*{Team}
+As a team, the project was started with five persons. Losing one person in during the first iteration, we had to carry out the work of one extra missing person. Because of this, the team quickly realized we had to maintain a clear organization. Making sure the organization of the project was defined using clear documents and following SCRUM we could implement features relatively fast with minimum overhead. There were also no problems with work that wasn't done or incomplete so we did not have to do work of the previous sprint in the following sprint. This resulted in the fact that the project was constant good on schedule.
```
Comment (doriandekoning) [inline deliverables/sprint5/exercise_3_essay.tex]: Is it me or is the first sentence a bit of a quirky sentence? Also should the we in the next sentence not be removed to match the style of the rest of the report?

**Comment by jochem725** [comment]
I've made some style changes to the essay, let me know if it can be merged.

**Comment by bryanvanwijk** [comment]
I've fixed two small errors, I think it can be merged now.

**Comment by egedebruin** [comment]
I fixed a very small typo and think it is ready for merging.

**Comment by doriandekoning** [comment]
Seems fine you can merge it.

**Pull request merged**